docs/go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/nginxinc/kubernetes-ingress/docs
 
 go 1.15
 
-require gitlab.com/f5/nginx/controller/poc/f5-hugo v0.24.1 // indirect
+require (
+	github.com/nginxinc/kubernetes-ingress v1.12.5 // indirect
```

Comment:
why is it importing version `1.12` of the Ingress Controller? 🤔

Reply:
it was imported with `latest` and github provided 1.12. It can be forced to a specific version